test/articles.test.js

```diff
@@ -43,6 +43,53 @@ describe('Integration tests for the article controller', () => {
+        title: 'The brow fox',
+        body: 'so i saw a dog',
+        description: 'narrative',
+        image: 'https://someimage.uplodersite.com',
+      };
+      const response = await chai.request(app).post('/api/v1/articles')
+      .set('x-access-token', myToken).send(articleDetails);
+        expect(response.status).to.equal(201);
+        expect(response.body).to.have.property('articleCreated');
+        expect(response.body.articleCreated).to.have.property('title');
+        expect(response.body.articleCreated.title).to.equal(articleDetails.title);
+        expect(response.body.articleCreated).to.have.property('body');
+        expect(response.body.articleCreated.body).to.equal(articleDetails.body);
+        expect(response.body.articleCreated).to.have.property('description');
+        expect(response.body.articleCreated.body).to.equal(articleDetails.body);
+        expect(response.body.articleCreated).to.have.property('readTime');
+        expect(response.body.articleCreated.readTime).to.equal('less than 1min');
+        expect(response.body.articleCreated).to.have.property('image');
+        expect(response.body.articleCreated.image).to.equal(articleDetails.image);
+        expect(response.body.articleCreated.isDraft).to.equal(true);
+    });
+    it('should add calculated read time to the article', async () => {
+      const articleDetails = {
+        title: 'The brow fox',
+        body: 'Knowing that millions of people around the world would be watching in person and on television and expecting great things from him — at least one  that millions of people around the world would be watching in person and on television and expecting great things from him — at least one more gold medal for America, if not another world record — during this, his fourth and surely his last appearance in the World Olympics, and realizing that his legs could no longer carry him down the runway with the same blazing speed and confidence in making a huge, eye-popping leap that they were capable of a few years ago when he set world records in the 100-meter dash and in the 400-meter relay and won a silver medal in the long jump, the renowned sprinter and track-and-field personality Carl Lewis, who had known pressure from fans and media before but never, even as a professional runner, this kind of pressure, made only a few appearances in races during the few months before the Summer Olympics in Atlanta, Georgia, partly because he was afraid of raising expectations even higher and he did not want to be distracted by interviews and adoring fans who would follow him into stores and restaurants demanding autographs and photo-opportunities, but mostly because he wanted to conserve his energies and concentrate, like a martial arts expert, on the job at hand: winning his favorite competition, the long jump, and bringing home another Gold Medal for the United States, the most fitting conclusion to his brilliant career in track and field.',
```

Comment:
Good job @ah-med, for making your implementation as concise as possible. However, it will be good we keep to the agreed convention of a maximum line length of 80 characters. You may consider breaking the lines and using string concatenation for articleDetails.body.
```suggestion
body: 'long text, 70+ characters'
+ 'long text 70+ characters',
```